script/draft-release/run.ts

```diff
@@ -0,0 +1,108 @@
+import { sort as semverSort, SemVer } from 'semver'
+
+import { spawn } from '../changelog/spawn'
+import { getLogLines } from '../changelog/git'
+import {
+  convertToChangelogFormat,
+  getChangelogEntriesSince,
+} from '../changelog/parser'
+
+import { Channel } from './channel'
+import { getNextVersionNumber } from './version'
+
+const jsonStringify: (obj: any) => string = require('json-pretty')
+
+async function getLatestRelease(excludeBetaReleases: boolean): Promise<string> {
```

Comment:
Functions that accept a boolean to conditionally do things are sometimes hard to read. Can we extract this into an enum?

```ts
export enum ReleaseOption {
  IncludeBeta,
  ExcludeBeta
}
```

That way, the function call reads better and doesn't require one to look at its definition to figure out what the bool does. ex. `getLatestRelease(ReleaseOption.IncludeBeta)`

Reply:
or `getLatestRelease({ excludeBetaReleases: false })`